### DISCUSSION TITLE
ImGui::Combo dropdown and selecteditem changed events

### DISCUSSION THREAD

**Discussion by aabilityuk** [ocornut/imgui#3911] [Old/Archived discussions]
Hi guys, i am very new to imgui the second day of burn. Could you please help me to find out the way how to listen and handle dropdown and selecteditem changed events on ImGui::Combo???

I create combo like this:
`static const char* items[]{"One","Two","three"};
    static int Selecteditem = 0;
    ImGui::Combo("MyCombo", &Selecteditem, items, IM_ARRAYSIZE(items));`

**Comment by ocornut** [comment]
Every function return `true` when something has changed.

**Reply by aabilityuk** [reply]
Thank you very much for reply. I rewrite the code like this, but the check value doesn't become true:
 ```
   static const char* items[]{"One","Two","three"};
    static int Selecteditem = 0;
    bool check =  ImGui::Combo("MyCombo", &Selecteditem, items, IM_ARRAYSIZE(items));
    if (check)
    {
    }
```

**Comment by aabilityuk** [comment]
I was able to recieve selected item changed event this way:

```
static const char* items[]{"One","Two","three"};
    static int Selecteditem = 0;
    if (ImGui::Combo("MyCombo", &Selecteditem, items, IM_ARRAYSIZE(items)))
    {
        // Here event is fired
    }
```
But i cant find a way how to handle the dropdown event when you push the arrow, help me please to find a solution

**Reply by rokups** [reply]
There is no event when you merely open dropdown.
